src/utilities/simIncludeGravBody.py

```diff
@@ -15,399 +14,496 @@
+        identifier="sun",
+        planetName="sun_planet_data",
+        displayName="sun",
+        modelDictionaryKey="",
+        mu=1.32712440018e20,
+        radEquator=695508000.0,
+        spicePlanetFrame="IAU_sun",
+    ),
+    "mercury": BodyData(
+        identifier="mercury",
+        planetName="mercury_planet_data",
+        displayName="mercury",
+        modelDictionaryKey="",
+        mu=0.022032e15,
+        radEquator=2439700.0,
+        spicePlanetFrame="IAU_mercury",
+    ),
+    "venus": BodyData(
+        identifier="venus",
+        planetName="venus_planet_data",
+        displayName="venus",
+        modelDictionaryKey="",
+        mu=3.24858599e14,
+        radEquator=6051800.0,
+        spicePlanetFrame="IAU_venus",
+    ),
+    "earth": BodyData(
+        identifier="earth",
+        planetName="earth_planet_data",
+        displayName="earth",
+        modelDictionaryKey="",
+        mu=0.3986004415e15,
+        radEquator=6378136.6,
+        spicePlanetFrame="IAU_earth",
+    ),
+    "moon": BodyData(
+        identifier="moon",
+        planetName="moon_planet_data",
+        displayName="moon",
+        modelDictionaryKey="",
+        mu=4.902799e12,
+        radEquator=1738100.0,
+        spicePlanetFrame="IAU_moon",
+    ),
+    "mars": BodyData(
+        identifier="mars",
+        planetName="mars_planet_data",
+        displayName="mars",
+        modelDictionaryKey="",
+        mu=4.28283100e13,
+        radEquator=3396190.0,
+        spicePlanetFrame="IAU_mars",
+    ),
+    "mars barycenter": BodyData(
+        identifier="mars barycenter",
+        planetName="mars barycenter_planet_data",
+        displayName="mars barycenter",
+        modelDictionaryKey="",
+        mu=4.28283100e13,
+        radEquator=3396190.0,
+        spicePlanetFrame="IAU_mars",
+    ),
+    "jupiter barycenter": BodyData(
+        identifier="jupiter barycenter",
+        planetName="jupiter barycenter_planet_data",
+        displayName="jupiter",
+        modelDictionaryKey="",
+        mu=1.266865349093058e17,
+        radEquator=71492000.0,
+        spicePlanetFrame="IAU_jupiter",
+    ),
+    "saturn": BodyData(
+        identifier="saturn",
+        planetName="saturn barycenter_planet_data",
+        displayName="saturn",
+        modelDictionaryKey="",
+        mu=3.79395000e16,
+        radEquator=60268000.0,
+        spicePlanetFrame="IAU_saturn",
+    ),
+    "uranus": BodyData(
+        identifier="uranus",
+        planetName="uranus barycenter_planet_data",
+        displayName="uranus",
+        modelDictionaryKey="",
+        mu=5.79396566e15,
+        radEquator=25559000.0,
+        spicePlanetFrame="IAU_uranus",
+    ),
+    "neptune": BodyData(
+        identifier="neptune",
+        planetName="neptune barycenter_planet_data",
+        displayName="neptune",
+        modelDictionaryKey="",
+        mu=6.83509920e15,
+        radEquator=24764000.0,
+        spicePlanetFrame="IAU_neptune",
+    ),
```

Comment:
Python allows trailing commas in argument lists:
```
test = [0, 1, 2, ] # ok
```
Personally I prefer to leave them there since you can easily add elements to the enumeration by copy-pasting an existing entry and the comma will already be there. If the standard in BSK is to not have trailing commas, Ill be happy to remove it.

Reply:
Thanks for the explanation.  I wasn't aware that this was a feature.